src/briefcase/platforms/linux/__init__.py

```diff
@@ -66,10 +66,17 @@ def support_package_url(self, support_revision):
         System packages don't use a support package; this is defined by
         the template, so this method won't be invoked
         """
+        python_download_arch = self.tools.host_arch
+        # use a 32bit Python if using 32bit Python on 64bit hardware
+        if self.tools.is_32bit_python and self.tools.host_arch == "aarch64":
+            python_download_arch = "armv7"
```

Comment:
Standalone Python fully supports creating armv6 redistributables...but they aren't included with the releases...only in CI...